### ISSUE TITLE
description: add "German"

### ISSUE THREAD

**Issue by mvhulten** [Sw24Softwares/StarkeVerben#14]
From the description in F-Droid:
> StarkeVerben is an Android application that aims to help people learning their strong verbs.

It would be useful if the description is updated to make clear that it is for learning *German* strong verbs.

**Comment by Swarthon** [comment]
Sorry for having been so long but we were a little busy.
In fact, we already have mentioned "German" in F-Droid. From the main description : 
> Learn your German strong verbs
Nonetheless, we wanted to add some screenshots to the F-Droid description, we could use this occasion to modify the description.

**Comment by mvhulten** [comment]
You are right, I overlooked this, so I will close this issue (minor suggestion near the bottom, though).

Possibly, I overlooked it because of the behaviour of F-Droid.  When I search for the program, it shows the name + short description
> Starke Verben
> Learn your German strong verbs

Then when I open it, it shows the name + long description
> Starke Verben
> StarkeVerben is an Android application that aims to help people learning their strong verbs.

and that does not have *German* in it; the term could be added in the long description as well.
In any case, it is not a big issue.  Probably, also not in F-Droid as repeating the short description is redundant and usually not needed.
